src/main/java/mekanism/common/network/to_server/PacketRobit.java

```diff
@@ -24,7 +24,8 @@ public class PacketRobit implements IMekanismPacket {
 
     private static final Map<String, List<IRobitSkinProvider>> EASTER_EGGS = Map.of(
             "sara", getPrideSkins(RobitPrideSkinData.TRANS, RobitPrideSkinData.LESBIAN),
-            "agnor", getPrideSkins(RobitPrideSkinData.GAY)
+            "agnor", getPrideSkins(RobitPrideSkinData.GAY),
+            "allay", List.<IRobitSkinProvider>of(MekanismRobitSkins.ALLAY)
```

Comment:
I think this is unnecessary. Given at a certain point having name just go to the skin itself makes things really not be an *easter egg*